packages/api/src/routes/medical/internal-organization.ts

```diff
@@ -56,38 +56,19 @@ router.put(
       cwApproved: orgDetails.cwApproved,
     };
     let org: Organization;
-    let orgCurrentActive = false;
     if (orgDetails.id) {
-      const currentOrg = await getOrganizationOrFail({ cxId, id: orgDetails.id });
-      orgCurrentActive = currentOrg.cqActive;
+      await getOrganizationOrFail({ cxId, id: orgDetails.id });
```

Comment:
Keep loading the Org even though its not needed for the update for Authorization purposes. Should we add a comment?